app/controllers/api/v3/schools_controller.rb

```diff
@@ -16,7 +16,7 @@ def show
 
       def schools_query(conditions: {})
         conditions[:lead_provider] = current_lead_provider
-        conditions[:contract_period_id] = contract_period&.id
+        conditions[:contract_period_years] = contract_period&.id
```

Comment:
should we use `contract_period_year` rather than `contract_period_years` as schools only accepts a single contract year 🤔 ?